state/session/reducer.js

```diff
@@ -1,7 +1,7 @@
 import * as types from './types';
 
 const initialState = {
-  session_id: null,
+  session_id: '',
```

Comment:
Use camelCase for variables, keys, function names, you also can enable this rule it in the eslint config

Reply:
Done.